harmonisation/specimen/license.md

```diff
@@ -14,9 +14,14 @@ For DWCA we will look at matches on two terms:
 If these terms are not available or are null we will leave this field empty.
 
 ### ABCDEFG
-Within ABCDEFG the license is not part of the unit level but is part of the dataset level.
-Within ABCDEFG data we will also look at two possible places:
+Within ABCDEFG the license can be part of both the unit level and the dataset level.
+We will first look at the unit level as it is the more precise level.
```

Comment:
I understand the choice, but the BioCASe Provider Software insists users to add a license to the metadata as this is an obliged element, thus it would not work without the element. Might be that people then do not match the license anymore  in the unit data, might be wrong but something to monitor.

Reply:
Ok yes, I would still say that you can put one at metadata level and then if you want a specific other one for one or more specimen you put it at unit level. I think the current logic should be sound, but let's monitor it